### DISCUSSION TITLE
srt to restream without obs? with encoding box type s2 kiloview

### DISCUSSION THREAD

**Discussion by esquilo444** [datarhei/restreamer#473] [Q&A]
hello, 

I want to know if it is possible to use a means other than obs to send an SRT stream to RESTREAMER?
Type s2 box from kiloview .. or other.
I already use the SRT with these boxes without any problem, but I can't make the link to RESTREAMER.
Thank you in advance for your light.
Cordially

jerome

**Comment by alexlealgalvao** [comment]
Hello everything is fine? I work with Liveu and use srt as Listener and Caller... see if you need to activate it as a caller on your equipment.
Hope this helps

**Comment by esquilo444** [comment]
Good morning,

yes, everything works perfectly, thank you again for your work and your availability.
For the moment I only use the server in listening mode...
this allows me not to struggle too much with the various remote sites and also works very well with 4G and 5G connections ... it works very well with my S2 from kiloview and also very well on other brand boxes linkpi...
everything is spot on..
Good day.

jerome


Envoyé à partir de Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________
